docs/releasenotes/release-1.113.0.md

```diff
@@ -0,0 +1,25 @@
+* Update operator to use server-side-apply (and stop using kubectl)
+
+* Initial support for status.observedState in ContainerCluster,
+  ContainerNodePool and RedisInstance; we are trying to encourage use of
+  `cnrm.cloud.google.com/state-into-spec: absent` and eventually
+  make it the default.  Some important resource information (such as the
+  certificate for connecting to a GKE cluster) is only currently available
+  in spec, and we recommend instead reading it from observedState
+  where this is available.  We expect to add more fields to observedState
+  in the future.
+
+* Isolate terraform provider into a git subtree so we can more directly fix
+  problems.
+
+* Special shout-outs to svwijk@, katrielt@, sofam@, higef@ for their
+  contributions to this release.
+
+## New Resources:
+
+* ComputeNetworkFirewallPolicy
+* TagsLocationTagBinding
+
+## New Fields:
+
+* connectorRef in RunJob (CloudRun Job)
```

Comment:
I forgot about the process, are we generating the notes by manually going over all the new commits?

https://github.com/GoogleCloudPlatform/k8s-config-connector/compare/v1.112.0...v1.113.0

`TagsLocationTagBinding` is an alpha resource. We could probably just following the existing release note format to avoid confusion?

```
* Added support for `ComputeNetworkFirewallPolicy` (v1beta1) resource.
* Added support for `TagsLocationTagBinding` (v1alpha1) resource.
```

Reply:
Used the language - thanks!   Were you thinking of _not_ splitting out the New Resources / New Fields headings.  I introduced this, thinking that this helps people read the notes better - do you think we should try this structure or go with a flat structure?

Reply:
Splitting out the New Resources / New Fields is more clear to me. 

I could be wrong, but I guess part of the old format came from running the script `scripts/generate-field-diffs-tf-upgrade/generate_field_changes.py` when updating Terraform prior to v4.84.0. e.g. https://cnrm-review.git.corp.google.com/c/cnrm/+/65900

Reply:
nit: For consistency, I would suggest using the same format for new fields.

Resource RunJob(v1beta1):
    Added spec.template.template.vpcAccess.connectorRef field.